modules/settings/assets/css/style.css

```diff
@@ -0,0 +1,39 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
```

Comment:
This one is blocking. Use `enqueue_scripts()` instead.

Reply:
I see that we already had, actually. So please just update the link in `modules/settings/module.php`